pkg/etcdenvvar/etcd_env.go

```diff
@@ -334,3 +352,8 @@ func getEtcdEndpoints(configmapLister corev1listers.ConfigMapLister, skipBootstr
 
 	return strings.Join(etcdURLs, ","), nil
 }
+
+func GibibytesToBytesString(dbSize int64) string {
```

Comment:
This doesn't need to be exported

Reply:
I just copied from the reveiw 😁😁

Ok will change it